pkg/graph/resolvers/trb_request_form_test.go

```diff
@@ -73,9 +89,23 @@ func (s *ResolverSuite) TestCreateTRBRequestForm() {
 		s.True((*updatedForm.CollabDateOther).Equal(collabDateOther))
```

Comment:
See my comment above about preferring `s.Equal()` over `s.True()`.

Reply:
The dates need to use the Equal method of time.Time to properly handle time zones - EqualValues produces an error on the CI server

Reply:
Ah - I didn't catch that these were times!

Yeah, I think with times using `.Equal` is good, because it accounts for any weirdness in the default timezone being changed in the objects (and just cares that they represent the correct time!)